tokio/tests/rt_handle_block_on.rs

```diff
@@ -388,6 +388,28 @@ rt_test! {
 
         rt.block_on(async { some_non_async_function() });
     }
+
+    #[test]
+    fn spawn_after_runtime_dropped() {
+        use futures::future::FutureExt;
+
+        let rt = rt();
+
+        let handle = rt.block_on(async move {
+            Handle::current()
+        });
+
+        let jh1 = handle.spawn(futures::future::pending::<()>());
+
+        drop(rt);
+
+        let jh2 = handle.spawn(futures::future::pending::<()>());
+
+        let err1 = jh1.now_or_never().unwrap().unwrap_err();
+        let err2 = jh2.now_or_never().unwrap().unwrap_err();
```

Comment:
@carllerche The `unwrap()` on `err1` here sometimes fails with the threaded scheduler. Isn't the drop call supposed to wait until the runtime is fully shut down?

Reply:
This is fixed by the "Fix loom test" commit